### PR TITLE
Game Mode

### PR DESCRIPTION
**Pull request by ratenbuuren**
Closes: #46

Implement a proper game mode with player setup (e.g. amount and controls) and victory state

- [x] Allow setup of players
- [x] Use chosen setup
- [x] Victory state (set winner in victory text)

### CODE REVIEW

File: Assets/Plugins/Editor/JetBrains/Unity3DRider.cs [hunk 1]
```diff
@@ -1,7 +1,7 @@
 //------------------------------------------------------------------------------
 // <auto-generated>
 //     This code was generated by a tool.
-//     Version: 2.0.0.0
+//     Version: 2.0.2.2395
```
Comment (kdhageman) [inline Assets/Plugins/Editor/JetBrains/Unity3DRider.cs:4]: I'm just wondering why the Rider changes are added? On purpose of by accident?

File: Assets/Scripts/GameManager.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 public class GameManager : MonoBehaviour {
     public static GameManager instance;
-    public bool gameOver = false;
+    public bool GameOver;
```
Comment (kdhageman) [inline Assets/Scripts/GameManager.cs]: Should this remain `public`? This enables someone to start the game with the `GameOver` field set as true.

**Review by kdhageman — CHANGES_REQUESTED** [review]
Looking good! Right now, the `BattleArena` scene is not working by default; for testing purposes this would be good to have (instead of selecting the number of players in the main menu each time).

File: Assets/Plugins/Editor/JetBrains/Unity3DRider.cs (hunk 1 above)
Comment (ratenbuuren) [reply Assets/Plugins/Editor/JetBrains/Unity3DRider.cs:4]: Update Rider to 2017.1 (commit message)

**Review by kdhageman — APPROVED** [review]

**Pull request merged**